docs/sources/features/explore/index.md

```diff
@@ -51,6 +51,68 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 You can close the newly created query by clicking on the Close Split button.
 
+## Query history
+
+> BETA: Query history is a beta feature.
+
+Query history is a list of queries that you have used in Explore. The history is local to your browser and is not shared with others. To open and interact with your history, click the **Query history** button in Explore.
+
+### View query history
+
+Query history lets you view the history of your querying. For each individual query, you can:
+
+- Replace Explore query for selected query by clicking on the query card.
+- Create and/or edit comment.
+- Copy query to clipboard.
+- Copy URL link with the query to clipboard.
+- Star query.
+
+### Manage favorite queries
+
+All queries that have been starred in **Query history** tab by clicking on star icon are displayed in **Starred** tab as well. This allows you to access your favourite queries faster, and to reuse these queries without typing them from scratch.
+
+### Sort query history
+
+By default query history shows you the most recent queries. This can be changed and you can sort your history in **Query history** and **Starred** tab by date or by data source name in ascending or descending order.
+
+1. Click the **Sort queries by** field.
+2. Select one of the following options:
+   - Time ascending
+   - Time descending
+   - Data source A-Z
+   - Data source Z-A
+
+> Note: If you are in split mode, chosen sorting mode is applied only to currently active panel.
+
+### Filter query history
+
+Filter query history in **Query history** and **Starred** tab by data source name:
+
+1. Click the **Filter queries for specific data source(s)** field
+2. Select data source for which you would like to filter your history. You can select multiple data sources.
+
+In **Query history** tab is is also possible to filter queries by date using the slider:
+
+1. Use vertical slider to filter queries by date.
```

Comment:
Please convert this list to a bulleted/unordered list. The sequence of the items is not important, so it should be unordered.